common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -253,6 +254,15 @@ def _check_scores_and_page_text(self, problem_scores, section_score, text):
         self.assertEqual(self._get_section_score(), section_score)
         self.assertTrue(self.progress_page.text_on_page(text))
 
+    def _check_for_sr_text_in_chart(self):
+        """
+        Ensures we have at least one .sr text in the graph.
+        We can assume if there's one, the JS is working correctly.
```

Comment:
I'm not sure why we can assume that the presence of one sr text means that they are all correct. I would prefer that the sr text is returned (as an array), and we assert that the expected values are present.